packages/now-next/src/index.ts

```diff
@@ -75,6 +75,7 @@ interface BuildParamsType extends BuildOptions {
   meta: BuildParamsMeta;
 }
 
+// bump for now/routing-utils
 export const version = 2;
```

Comment:
You don't need a PR for this you can do this:

```
yarn publish-canary --force-publish=@now/routing-utils,@now/next
```

See [the wiki](https://github.com/zeit/now/wiki/Creating-a-Release).

Reply:
Oh nice, thanks!